spec/model_renamer/migration_generator_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe MigrationGenerator do
+  after(:all) do
+    FileUtils.rm_rf('./db')
+  end
```

Comment:
Open to suggestions here. Got some strange errors when I tried to use MemFs here, so I let rails do its thing and create an actual migration file and then delete it at the end.

Reply:
Interesting.. Should this just be an `after(:each)`?